nixos/modules/services/x11/desktop-managers/plasma5.nix

```diff
@@ -184,15 +183,12 @@ in
 
   config = mkMerge [
     (mkIf cfg.enable {
-      services.xserver.desktopManager.session = singleton {
-        name = "plasma5";
-        bgSupport = true;
-        start = startplasma;
```

Comment:
Since we are removing the custom session script, we should also remove `startplasma` above, but it calls several scripts which must still run at start-up.

Reply:
Now startplasma is called in services.xserver.displayManager.setupCommands